### DISCUSSION TITLE
Not able to start sensor on eval installation

### DISCUSSION THREAD

**Discussion by tslvr3** [Security-Onion-Solutions/securityonion#5129] [Unsupported Versions]
How to start senor and steno service on security onion 2.3.61

**Comment by weslambert** [comment]
Does so-status show it as not running? You should be able to do something like the following:

`sudo so-pcap-start`

**Reply by tslvr3** [reply]
Hello weslambert,

Thank you for the response.

it's showing status as **starting** always but it was not started even after 48hr..

Still the status is same..

I will use the given command and let you know the status.

Thanks & Regards
Ramana T

**Reply by tslvr3** [reply]
Hello weslambert,

Now steno is working thank you. Could you please let me know how to start sensoroni service..

Thanks & Regards
Ramana T

**Comment by dougburks** [comment]
Something doesn't seem right. You shouldn't have to manually start these services. Does your hardware meet the requirements?
https://securityonion.net/docs/hardware

After verifying hardware, I'd recommend a fresh installation of the latest Security Onion 2.3.70:
https://blog.securityonion.net/2021/08/security-onion-2370-now-available.html

**Reply by tslvr3** [reply]
Hello DougBurks,

Thank you for the response.

I will do fresh installation using new iso (2.3.70) and reach you if anything required...

Thanks & Regards
Ramana T
